src/popup/components/Login/index.tsx

```diff
@@ -44,34 +44,23 @@ const InputContainer = styled.div`
 type LoginProps = {
   loading: boolean;
   onNext: (password: string) => void;
-  error: string;
+  error?: string;
 };
 
 function Login(props: LoginProps) {
-  const { loading, onNext, error: propError } = props;
+  const { loading, onNext, error } = props;
 
   const [password, setPassword] = useState("");
-  const [error, setError] = useState(propError);
+  const [clearError, setClearError] = useState(true);
 
-  const onClick = () => {
-    if (loading || isPasswordEmpty || hasError) {
-      return;
-    }
+  const valid = password.length > 0 && (clearError || error == null);
```

Comment:
```suggestion
  const valid = password.length >= 8 && (clearError || error == null);
```

Since we define `const MIN_LENGTH = 8;` elsewhere, I feel this might make more sense. Better yet would be not to hardcode this 8 but to read it from the same source.